project_script/main.py

```diff
@@ -1,108 +1,105 @@
-import os.path
-import numpy as np
 import pandas as pd
 import matplotlib.pyplot as plt
 from scipy import integrate
 import ask
-
-
-def separate_even_columns(data_frame):
-    return data_frame.iloc[:, [i for i in range(len(data_frame.columns)) if i % 2 == 1]]
-
-
-def separate_odd_columns(data_frame):
-    return data_frame.iloc[:, [i for i in range(len(data_frame.columns)) if i % 2 == 0]]
-
-
-def is_valid_file(file_path):
-    return True if file_path != "" and os.path.exists(file_path) else False
+import datatreatment as dt
+import constantvalues as cv
 
 
 def message_invalid_path():
     return print("Arquivo inválido ou inexistente \n")
 
 
-def is_bigger_data_frame(data_frame_1, data_frame_2):
-    return len(data_frame_1) > len(data_frame_2)
-
-
-def equalize_data_frame_rows(data_frame_1, data_frame_2):
-    return data_frame_1.loc[0:len(data_frame_2)-1, :]
-
-
 def sanitize_data_frames():
     global even_columns_subtracted, time_photo_celiv_ramp_time
-    if is_bigger_data_frame(time_photo_celiv_ramp_time, even_columns_subtracted):
-        time_photo_celiv_ramp_time = equalize_data_frame_rows(time_photo_celiv_ramp_time, even_columns_subtracted)
+    if dt.is_bigger_data_frame(
+            time_photo_celiv_ramp_time,
+            even_columns_subtracted
+    ):
```

Comment:
If this line doesn't cross the limit line pattern, then doesn't need to break. But if it's do, ok, NP.

Reply:
Done!